app/controllers/application_controller.rb

```diff
@@ -9,17 +10,17 @@ class ApplicationController < ActionController::Base
   protect_from_forgery
   before_action :app_setup_check
   before_action :authenticate_user!, unless: :skip_authentication?
-  before_action :cart, unless: :devise_controller?
 
-  with_options unless: ->(_u) { User.count == 0 } do |c|
-    c.before_filter :load_configs
-    c.before_filter :seen_app_configs
-    c.before_filter :fix_cart_date
-    c.before_filter :set_view_mode
-    c.before_filter :check_view_mode
-    c.before_filter :make_cart_compatible
```

Comment:
We're still testing this - how is that passing if you removed the `before_filter`?